src/librawspeed/decoders/MrwDecoder.h

```diff
@@ -31,10 +31,10 @@ namespace RawSpeed {
 class CameraMetaData;
 class TiffIFD;
 
-typedef struct {
+using mrw_camera_t = struct mrw_camera_t {
```

Comment:
I suggest to use 'standard' c++ here instead of a `using` fix of some c-code that should have been ported to c++ the time it was first introduced into rawspeed. Simply write
```
  struct mrw_camera_t {
    ...
  };
```